app/components/publications/publication/proofs/proof-upload-modal.js

```diff
@@ -27,12 +27,35 @@ export default class PublicationsPublicationProofsProofUploadModalComponent exte
     this.initValidation();
   }
 
+  get isLoading() {
+    return this.cancel.isRunning || this.save.isRunning;
+  }
+
+  get isCancelDisabled() {
+    return this.cancel.isRunning || this.save.isRunning;
+  }
+
   get isSaveDisabled() {
-    return !this.file || !this.validators.areValid;
+    return !this.file || this.file.isDeleted || !this.validators.areValid;
+  }
+
+  @task({
+    drop: true,
+  })
```

Comment:
Louter ter info, wat er staat is correct en moet niet gewijzigd worden: er is ook een `@dropTask` decorator in [ember-concurrency-decorators](https://github.com/machty/ember-concurrency-decorators#available-decorators)